Build/validate-qir.ps1

```diff
@@ -57,23 +71,24 @@ function Build-One {
 
 # The commented out lines are sample projects that are not yet compatible for QIR generation/execution. 
 # 'not compatible' means that the structure of the sample is not compatible for QIR generation.
-# 'needs argument(s)' means that the sample can generated QIR, but running the exe requires command line arguments.
+# 'relies on hardware' means that the sample targets a hardware architecture that is not yet supported for QIR.
+# 'memory leak' means that the sample stalls when running and uses way too many system resources.
```

Comment:
Is there a bug filed that tracks these memory leaks? Would be good to link in comments so that there's an actionable path to adding affected samples back into validation.

Reply:
Yes this issue here: https://github.com/microsoft/qsharp-runtime/issues/757. I've added this link to the comments, per your suggestion.